syedsaif/HTML/d2.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <div style="background-color:lightblue">
+        <h1 style="text-align: center;">Google</h1>
+        <div><em>This article is about the company.</em></div>
+             <b>From the garage to the Googleplex </b><br>
+             <p>The Google story begins in 1995 at <b>Stanford University</b>.<br>
+             <b><i>Larry Page</i></b> was considering Stanford for grad school and <b><i>Sergey Brin</i></b>, a student
+        there, was assigned to show him around.</p><br>
+        <div>
+        <p>By some accounts, they disagreed about nearly everything during that first meeting, but by the following
+            year they struck a <mark>partnership</mark>.<br>
+            Working from their dorm rooms, they built a search engine that used links to determine the importance of
+            individual pages on the <strong style="color:blue;"> World Wide Web. </mark>.<br>
+            They called this search engine <mark style="background-color:cyan;">Backrub</mark>.</br><hr>
+        </div>
+        <b>Initial public offering</b><br>
+        <p>Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+            At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 30 20
+            years, until the year 2024.</p>
+            <div style="text-align:center;"><b style="color: blue;">Thank You!</b></div>
+     </head>
+</html>
```

Comment:
missing body tag

Reply:
i have modified the file,kindly check and verify the same.